components/camel-huawei/camel-huaweicloud-obs/src/main/java/org/apache/camel/component/huaweicloud/obs/OBSEndpoint.java

```diff
@@ -0,0 +1,278 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.huaweicloud.obs;
+
+import com.obs.services.HttpProxyConfiguration;
+import com.obs.services.ObsClient;
+import com.obs.services.ObsConfiguration;
+import org.apache.camel.Category;
+import org.apache.camel.Consumer;
+import org.apache.camel.Processor;
+import org.apache.camel.Producer;
+import org.apache.camel.component.huaweicloud.obs.models.OBSRegion;
+import org.apache.camel.component.huaweicloud.obs.models.ServiceKeys;
+import org.apache.camel.spi.Metadata;
+import org.apache.camel.spi.UriEndpoint;
+import org.apache.camel.spi.UriParam;
+import org.apache.camel.spi.UriPath;
+import org.apache.camel.support.DefaultEndpoint;
+import org.apache.camel.util.ObjectHelper;
+
+/**
+ * To provide stable, secure, efficient, and easy-to-use cloud storage service on Huawei Cloud
+ */
+@UriEndpoint(firstVersion = "3.11.0", scheme = "hwcloud-obs", title = "Huawei Cloud Object Storage Service (OBS)",
+             syntax = "hwcloud-obs:operation",
+             category = { Category.CLOUD })
+public class OBSEndpoint extends DefaultEndpoint {
+
+    @UriPath(description = "Operation to be performed", displayName = "Operation", label = "producer", secret = false)
```

Comment:
no need to add `secret=false`, it is by default `false`

Reply:
Done. Removed secret = false from all parameters with it